sunpy/extern/appdirs.py

```diff
@@ -28,11 +28,11 @@
 if sys.platform.startswith('java'):
     import platform
     os_name = platform.java_ver()[3][0]
-    if os_name.startswith('Windows'): # "Windows XP", "Windows 7", etc.
```

Comment:
get out of my extern/ autopep8

Reply:
Yeah I need to path exlude it later on. I will do it once we figure out what leave and what checks we want to leave in.